doc: improved description of how to program network sample using SES-NE

tejlmand opened the pull request:
This commit improves the current description on how to program the
network sample hci_rpmsg when using SES-NE for the nRF5340dk.

Signed-off-by: Torsten Rasmussen <Torsten.Rasmussen@nordicsemi.no>

b-gent commented:
@tejlmand I've added a commit with my review. Please have a look and squash before merging.

This was open extremely late to add it for the dev tag, we'll figure out tomorrow if we can still do it.

On my part, I am approving this now, as this seems like crucial information to have in an nRF53-focused release.

tejlmand commented:
> @tejlmand I've added a commit with my review. Please have a look and squash before merging.
 
Looks good and squashed. Fixed the reference issue where link was changed, but title was not updated.
Thanks for the update.

> This was open extremely late to add it for the dev tag, we'll figure out tomorrow if we can still do it.
> 

Yes I know.
It was not until verifying the latest SES J-Link with nRF53 updates that it became apparent that the current description was not sufficient for users.

> On my part, I am approving this now, as this seems like crucial information to have in an nRF53-focused release.

Exactly. And there is a task in improving this workflow in the future to give an even better user experience.
